src/assets/CollectibleDetectionController.test.ts

```diff
@@ -289,10 +298,14 @@ describe('CollectibleDetectionController', () => {
   });
 
   it('should detect, add collectibles and do nor remove not detected collectibles correctly', async () => {
+    const selectedAddress = '0x1';
     collectibleDetection.configure({
       networkType: MAINNET,
-      selectedAddress: '0x1',
+      selectedAddress,
     });
+    collectiblesController.configure({ selectedAddress });
```

Comment:
I know these are unit tests so we don't need to test the integration points with other Controllers but I generally like to use the upstream method where the piece of state we're subscribed to actually changes (in this case it would be preferences.setSelectedAddress(selectedAddress)). I think it makes our network of assumptions safer but I'm open to being convinced otherwise! Thoughts @Gudahtt @wachunei @gantunesr

Reply:
If we wanted to test this robustly we'd do both. I'd rather us migrate to using the new base controller than waste our time doing that though, because we don't really want `configure` to be public API like this.

Following your suggestion would be better at least in that it'd be more compatible with BaseControllerV2 when we do migrate.